### DISCUSSION TITLE
missing binary when using full install

### DISCUSSION THREAD

**Discussion by verdverm** [containerd/nerdctl#2325] [Q&A]
### Description

I have finally got rootless setup working, but still unable to get it to uninstall.

The root of my issue was the missing binaries from the `uidmap` package

saw errors like: 
```
FATA[0000] rootless containerd not running? (hint: use `containerd-rootless-setuptool.sh install` to start rootless containerd): environment variable XDG_RUNTIME_DIR is not set, see https://rootlesscontaine.rs/getting-started/common/login/
```

### Steps to reproduce the issue

1. Fresh install on a fresh OS
2. run rootless setup, should experience error
3. nerdctl version should error, complaining about uid/gid 
4. try to uninstall, still get errors
5. `sudo apt install uidmap`

commands start working

### Describe the results you received and expected

an easier time installing and getting setup

### What version of nerdctl are you using?

was unable to figure out the previous version, installed v1.4.0 today, so probably the 1.3.x before that iirc

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

### Host information

ChromeOS and default Debian linux app (bullseye)

### Other notes

The following https://github.com/containerd/nerdctl/blob/main/docs/faq.md#how-to-uninstall---cant-remove-localsharecontainerd does not match what the first step prints out, it tells you to run a shorter command with only one directory

There was also a `.config/buildkit/buildkit.toml` that had a rootless=true that was also impacting this, I had to set this to false to get some of the commands working while trying to debug / fix. Not sure of the order anymore... because I think it is still false, but now I am running nerdctl in rootless mode, so not sure what relation they actually have

**Comment by AkihiroSuda** [comment]
> I have finally got rootless setup working, but still unable to get it to uninstall.

https://github.com/containerd/nerdctl/blob/v1.4.0/docs/faq.md#how-to-uninstall---cant-remove-localsharecontainerd

```
containerd-rootless-setuptool.sh uninstall
rootlesskit rm -rf ~/.local/share/containerd ~/.local/share/nerdctl ~/.config/containerd
```
